test/dlc_tests/ecs/mxnet/training/test_ecs_mxnet_training.py

```diff
@@ -74,6 +74,7 @@ def test_ecs_mxnet_training_dgl_cpu(cpu_only, py3_only, ecs_container_instance,
     ecs_utils.ecs_training_test_executor(ecs_cluster_name, cluster_arn, training_cmd, mxnet_training, instance_id)
 
 
+@pytest.mark.skip(reason="Skip until DGL with cuda 11.0 is available")
```

Comment:
Please insert this skip inside the test function and check for `cu110` in the image tag, rather than just skipping for all GPU images, so that older framework versions can still be tested.